index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Joshua Ivie | Portfolio</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./assets/css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="shortcut icon" href="./assets/img/favicon.png">
+  <link rel="icon" href="./assets/img/favicon-32x32.png">
+  <link rel="apple-touch-icon" href="./assets/img/apple-icon.png">
+  <link rel="apple-touch-icon" sizes="72x72" href="./assets/img/apple-icon-72x72.png">
+  <link rel="apple-touch-icon" sizes="114x114" href="./assets/img/apple-icon-114x114.png">
+</head>
+
+<body>
+  <header>
+    <nav class="top-navigation">
+      <a href="index.html" class="logo">
+        <img src="./assets/img/logo.svg" alt="Joshua Ivie">
+      </a>
+      <div id="menuToggle" class="menu-toggle">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </nav>
+    <div class="hero">
+      <div class="hero-shapes"></div>
+      <div class="hero-content">
+        <h1 class="title">Hello There! 👋 <br /> Welcome to my Website!</h1>
+        <p class="introduction">I'm Joshua Ivie and I'm a designer and full stack engineer with 5+ years of experience.
+          Like anything you
+          see or just want to chat over a coffee? Here are a few links you can contact me
+          through.</p>
+        <h5 class="sub-title">Lets Connect</h5>

```

Comment:
- [ ] The button `Let's connect` should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)